app/decorators/physical_rack_decorator.rb

```diff
@@ -0,0 +1,24 @@
+class PhysicalRackDecorator < MiqDecorator
+  def fileicon
+    # TODO: change it once we have a file icon for a rack
+    nil
+  end
+
+  def self.fonticon
+    'pficon pficon-enterprise'
+  end
+
+  def quadicon(_n = nil)
+    icon = {
+      :top_left => {:text => (physical_servers ? physical_servers.size : 0)}
```

Comment:
Why the `0` at the end? My assumption is that the `physical_servers` method is not always available, or does something go wrong if it returns `nil˙?